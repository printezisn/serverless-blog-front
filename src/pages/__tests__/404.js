import React from "react"
import renderer from "react-test-renderer"

import NotFoundPage from "../404"

jest.mock("../../components/layout", () => "Layout")

describe("404 page", () => {
    it("renders correctly", () => {
        const notFoundPage = renderer.create(<NotFoundPage />)

        expect(notFoundPage.toJSON()).toMatchSnapshot()
    })
})
