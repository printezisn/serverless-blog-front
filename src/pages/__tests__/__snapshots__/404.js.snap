// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 page renders correctly 1`] = `
<Layout>
  <article
    className="has-text-centered"
  >
    <p
      className="has-text-warning"
    >
      <i
        className="fa fa-9x fa-exclamation has-text-warning"
      />
    </p>
    <header>
      <h2
        className="title is-2"
      >
        Page Not Found
      </h2>
    </header>
    <p>
      The page you requested was not found.
    </p>
  </article>
</Layout>
`;
